src/components/App.tsx

```diff
@@ -1,6 +1,9 @@
 import * as React from 'react';
 import { Fragment } from 'react';
-import Home from './Home';
+import CommentsForm from './Comments';
+import Signup from './SIGN_UP';
```

Comment:
import Signup from './Main/signup';
This has to point to the signup component file. The signup file is located in the Main directory in components.
It also appears there is a conflict in this file so you will have to resolve this too.